src/material/list/selection-list.ts

```diff
@@ -371,8 +402,7 @@ export class MatSelectionList
       .withHomeAndEnd()
       .withTypeAhead()
       .withWrap()
-      // Allow navigation to disabled items.
-      .skipPredicate(() => false);
+      .skipPredicate(_skipPredicate);
```

Comment:
Why did we have  to extract out a function just to return false?

Reply:
No, we do not need to extract out a function :). I only did that because it makes it easier to add a code comment. I can inline it instead if that's preferred, but I thought it would be easier to read the more detailed comment this way.

Reply:
I changed this to not extract the skipPredicate into a separate function.